pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
+    name = 'run'
+    usage = rwt.commands.help_doc
```

Comment:
Oh, not the rwt tests, but the pip tests check all commands to ensure that they have a viable docstring. I need to look into the docs situation.